core/src/test/java/io/undertow/util/SimpleObjectPoolTestCase.java

```diff
@@ -0,0 +1,53 @@
+package io.undertow.util;
```

Comment:
We are missing the license header and a javadoc with @author

Reply:
Updated